src/SortableContainer/index.js

```diff
@@ -339,12 +346,13 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 				if (transitionDuration) {
 					node.style[`${vendorPrefix}TransitionDuration`] = `${transitionDuration}ms`;
 				}
+
 				if (index > this.index && (sortingOffset + offset >= edgeOffset)) {
-					translate = -this.dimension;
+					translate = -this.dimensionWithMargins;
```

Comment:
@clauderic 
From [MDN's Mastering margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing)
> Top and bottom margins of blocks are sometimes combined (collapsed) into a single margin whose size is the largest of the margins combined into it, a behavior known as margin collapsing.

[Live example](http://jsfiddle.net/fhdjznwe/)

And I updated this PR to support this case.